src/middlewares/generateDocs.ts

```diff
@@ -0,0 +1,26 @@
+import path from 'path'
+import { logInfoMessage, logLineWithIcon, logError } from '../lib/logger.js'
+import Generator from '@asyncapi/generator'
+
+export default async (spec, config, resDir) => {
+  logInfoMessage(`Generating docs for your parsed specification...`)
+  const configData = config.generator
+  const resolvedData = spec._json
+  const generator = new Generator(
+    configData && configData.template
+      ? `@asyncapi/${configData.template}`
+      : '@asyncapi/markdown-template',
```

Comment:
Oh! After checking tests I realized about this. Don't scope templates to `@asyncapi`. People might want to use a template of their own:

```suggestion
    configData?.template || '@asyncapi/markdown-template',
```